src/views/spotlights/housing/dashboard.vue

```diff
@@ -1,46 +1,255 @@
 <template>
   <DashboardCard width="full">
     <template #subtitle>
-      This section will conceptualize what's on the page
-    </template>
-    <template #content>
-      What will users find? What information are they gathering?
+      <div class="content">
+        <p>
+          You can use this page to explore how housing affects COVID-19 outcomes
+          in Rhode Island. Where someone lives impacts their chances of getting
+          very sick with COVID-19. We found that people who lived in low-income
+          housing had a high chance of going to the hospital with COVID-19. We
+          also saw that these chances were especially high for older adults.
+        </p>
+        <h5>What kinds of housing exist in Rhode Island?</h5>
+        <div class="table-container m-auto">
+          <table class="table is-narrow is-bordered">
+            <tbody>
+              <tr>
+                <!-- <tr bgcolor="#C7B3F9"> -->
+                <th>Housing type</th>
+                <th>What it is</th>
+              </tr>
+              <tr>
+                <td>Public</td>
+                <td>
+                  Managed by government agencies to provide housing for low
+                  income individuals and families
+                </td>
+              </tr>
+              <tr>
+                <td>Section VIII</td>
+                <td>
+                  Similar purpose as public housing, but housing is privately
+                  owned and subsidized through government spending
+                </td>
+              </tr>
+
+              <tr>
+                <td style="white-space: pre">Non-Congregate</td>
+                <td>
+                  Privately owned or rented residence that is not subsidized
+                  through government spending
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Congregate
+                  <p style="font-size: 1rem">
+                    <em>*not included in this analysis</em>
+                  </p>
+                </td>
+                <td>
+                  Individuals typically not related to each other living in
+                  close proximity and/or sharing living facilities (e.g.,
+                  assisted living residences, nursing homes, group homes,
+                  college dormitories, correctional facilities)
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
     </template>
   </DashboardCard>
   <DashboardCard width="full">
     <template #content>
       <ControlPanel
```

Comment:
In the ControlPanel file, can you add in the class "label" to the Label there? It gives some extra spacing.

```vue
      <label :for="type" class="label is-family-primary"
        ><strong>{{ options.label }}</strong></label
      >
```